api/views.py

```diff
@@ -37,3 +38,25 @@ class QuestionCategoryViewSet(viewsets.ReadOnlyModelViewSet):
 class UserStatsViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = MyUser.objects.all()
     serializer_class = UserStatsSerializer
+
+
+class ActionsViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = ActionsSerializer
+
+    def get_queryset(self):
+        user = self.request.user
+        is_recent = self.request.query_params.get('recent', False)
```

Comment:
this parameter is not defined anywhere? I'm 99% sure that it doesn't show up in the documentation. Make sure you have used the proper format of the parameters here according to the django-restframework documentation.